lib/tests/testlib.c

```diff
@@ -78,9 +78,10 @@ build_pedigree_sim(msp_t *msp, tsk_table_collection_t *tables, gsl_rng *rng,
 
     /* Insert the pedigree individuals */
     for (j = 0; j < num_individuals; j++) {
-        ret = tsk_individual_table_add_row(&tables->individuals, 0, NULL, 0, NULL, 0,
-            /* encode the parents in the metadata */
-            (void *) (parents + j * ploidy), ploidy * sizeof(tsk_id_t));
+        ret = tsk_individual_table_add_row(
+            &tables->individuals, 0, NULL, 0, parents + j * ploidy, ploidy, NULL, 0);
+        /* NULL, 0, */
```

Comment:
I guess you're keeping this here for future ref? Maybe a note to that effect if that is the intent.

Reply:
Hah, no, I just missed it. Good spot.